CONTRIBUTING.md

```diff
@@ -72,6 +72,9 @@ fetch the QR code from API and display it on the Profile page (ProfileFragment.k
 fixes #1234
 ```
 A good rule of thumb for the commit message is to have a present tense verb, followed by whatever it is you're doing in as much detail as possible in 50 chars. Capitalize words correctly and follow general English.
+### For more details, visit
+- http://karma-runner.github.io/0.13/dev/git-commit-msg.html
```

Comment:
Please add links using [this](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links)
The name should be the article name
Git commit message guidelines (first link)
Writing Good Commit Messages: A Practical Git Guide (2nd link)
respectively

Reply:
I have made the changes.